app/controllers/stylesheets_controller.rb

```diff
@@ -23,9 +23,24 @@ def show
   def color_scheme
     params.require("id")
     params.permit("theme_id")
+    params.permit("request_dark")
 
     manager = Stylesheet::Manager.new(theme_id: params[:theme_id])
     stylesheet = manager.color_scheme_stylesheet_details(params[:id], fallback_to_base: true)
+
+    if params[:request_dark]
+      dark_href =
+        manager.color_scheme_stylesheet_link_tag_href(
+          params[:id],
+          dark: true,
+          fallback_to_base: false,
+        )
+      if dark_href
+        stylesheet = stylesheet.dup
+        stylesheet[:new_dark_href] = dark_href
+      end
+    end
```

Comment:
I feel like this logic belongs inside `Stylesheet::Manager` rather than the controller. Perhaps we add the `include_dark_scheme` option to `manager.color_scheme_stylesheet_details`?